app/templates/main/userProfile.html

```diff
@@ -294,6 +295,112 @@ <h6>{{bgType.description}}</h6>
     {% endif %}
     <!-- End Background Check Table -->
 </div>
+<!-- Start Notes section -->
+  <div class="accordion-item">
```

Comment:
This accordion item is not inside the main User Profile accordion so there is a small separation between the the Notes tab and the Background Check tab.

Reply:
fixed